app/models/instance/user_invitation.rb

```diff
@@ -2,8 +2,8 @@
 class Instance::UserInvitation < ApplicationRecord
   acts_as_tenant :instance, inverse_of: :invitations
 
-  before_validation :generate_invitation_key, if: :new_record?
-  before_validation :set_defaults, if: :new_record?
+  after_initialize :generate_invitation_key, if: :new_record?
```

Comment:
Some test cases do not save the object before accessing attributes, so they break when `before_validation` is used.